tests/openssl/enc/CMakeLists.txt

```diff
@@ -335,6 +345,10 @@ function (add_openssl_test_enc NAME BUILDTEST CRYPTO_LIB)
     list(APPEND INCLUDE_PATHS ${TEST_OPENSSL_DIR}/crypto/rsa)
   endif ()
 
+  if ("${NAME}" STREQUAL "rsa_sp800_56b_test")
```

Comment:
This looks like it was added by a merge conflict, we're adding the same rsa_sp800_56b_test twice